draft-ietf-quic-recovery.md

```diff
@@ -819,6 +819,16 @@ kGranularity:
 kInitialRtt:
 : The RTT used before an RTT sample is taken. The RECOMMENDED value is 100ms.
 
+kPacketNumberSpace:
+: An enum to enumerate the three packet number spaces.
+~~~
+  enum kPacketNumberSpace {
+    Initial,
+    Handshake,
+    Application data,
```

Comment:
It's an enum, so how about ApplicationData

OT: At some point there's a hope to make this pseudocode compile in Python.

Reply:
Python enums don't look particularly nice though: https://docs.python.org/3/library/enum.html#creating-an-enum

Reply:
I renamed the variable.